js/collaborator-agent/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "collab-ts",
+  "version": "0.0.0",
+  "license": "MIT",
+  "private": true,
+  "main": "dist/index",
+  "types": "dist/index",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "scripts": {
+    "clean": "npx rimraf ./dist",
+    "build": "tsc",
+    "start": "node .",
+    "dev": "npx nodemon -w \"./src/**\" -e ts --exec \"node -r ts-node/register -r dotenv/config ./src/index.ts\"",
+    "dev:teamsfx": "npx env-cmd --silent -f .env npm run dev",
```

Comment:
this looks old.....

Reply:
can you try creating a new project using `npx @microsoft/teams.cli@preview new` and making sure these match?